_sections/merge-conflicts-android-studio.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Android Studio
+tab: android
+---
+
+### Overview
+
+GitLive allows you to see your teammates’ changes to a file in the gutter of your editor and get notified of conflicts as soon as they occur, even across branches and uncommitted changes.
+
+![jetbrains Gutter](/uploads/jetbrains-gutter-cherry-pick.gif "jetbrains Gutter"){:class="screenshot"}
+### Gutter indicators for teammate’s changes
+GitLive adds gutter indicators to show you where your teammate’s have made changes to the file you have open, these update in real-time as you and your teammates are editing.
+
+Insertions are represented by a green arrow, modifications are denoted by a blue line and deletions by a  grey line.
+
+![Android Studio Gutter](/uploads/android-studio-gutter-teammate.jpeg "Android Studio Gutter"){:class="screenshot"}
+
+### Conflict Indicators
+If your teammate’s changes are conflicting with your local changes, they are accompanied by the bright red conflict indicator. These conflicts can be uncommitted local changes you have not pushed yet or existing changes on your branch which are conflicting with their branch and/or uncommitted changes.
```

Comment:
These conflicts can be uncommitted local changes you have not pushed yet or existing changes on your branch which that conflict with your teammates' changes.